UI/PlayerListWnd.cpp

```diff
@@ -818,8 +844,9 @@ void PlayerListWnd::Update() {
     }
 }
 
-void PlayerListWnd::Refresh() {
+void PlayerListWnd::Refresh(bool clean_empire_status) {
```

Comment:
what's the purpose of the `clean_empire_status` parameter?

Reply:
On new game and new turn empire status must be cleared.

Reply:
Why not always?

Reply:
Because players list also updates when player enters/leaves the game. It shouldn't change empire status.

Reply:
Would it change the status if the player enters/leaves? Is a non-connected empire not considered waiting? Seems like an arbitrary special case...

Reply:
Currently if player enters game empire become unready because I think if player enters game, then he wants to change something, but it could be changed.
If player leaves game or empire is non-connected then status leaves unchanged. Moreover status is saved in the savegames end restored from them.
Only player or new turn can make empire status unready, and only player can make empire status ready.